src/main/java/classify/user/InputParsing.java

```diff
@@ -479,6 +488,13 @@ public static LocalDate readInDate(Scanner in) {
 
     
     //@@author Cryolian
+    /**
```

Comment:
Good Javadocs, the first of its kind!
![image](https://github.com/AY2324S2-CS2113-T13-3/tp/assets/110646458/4078f89c-58f9-4713-bb82-57c467c90a28)